### PR TITLE
Added shuhaab.json TXT Record

### PR DESCRIPTION
**Pull request by Shuhaab-coder**
I added the record now Can I get The domain

### CODE REVIEW

File: domains/shuhaab.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "subdomain": "shuhaab",
+    "domain": "is-a.dev",
```
Comment (Stef-00012) [inline domains/shuhaab.json:3]: this is not neeed

File: domains/shuhaab.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "records": {
```
Comment (Stef-00012) [inline domains/shuhaab.json:4]:
```suggestion
    "record": {
```

File: domains/shuhaab.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+      "TXT": ["vc-domain-verify=shuhaab.is-a.dev,4f90144524ddbcf1a09b"]
```
Comment (Stef-00012) [inline domains/shuhaab.json:5]: this goes on a different `_vercel.shuhaab.json` file, see https://docs.is-a.dev/vercel

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/shuhaab.json (hunk 1 above)
Comment (Shuhaab-coder) [reply domains/shuhaab.json:3]: okay

**Pull request closed without merging**